tests/db_engine_specs/postgres_tests.py

```diff
@@ -430,11 +430,12 @@ def test_base_parameters_mixin():
         "port": 5432,
         "database": "dbname",
         "query": {"foo": "bar"},
+        "encryption": True,
     }
     sqlalchemy_uri = PostgresEngineSpec.build_sqlalchemy_uri(parameters)
-    assert (
-        sqlalchemy_uri
-        == "postgresql+psycopg2://username:password@localhost:5432/dbname?foo=bar"
+    assert sqlalchemy_uri == (
+        "postgresql+psycopg2://username:password@localhost:5432/dbname?"
+        "foo=bar&sslmode=verify-ca"
```

Comment:
are there specific types of sslmodes? we could go with an enum approach if the list is small enough

Reply:
@hughhhh the user is not choosing one, with the new form they only choose between SSL on or off. If they choose "on" we set it to `verify-ca`, since it's the least restrictive one that still ensures an encrypted connection (note that `require` is identical to `verify-ca`).

If the user wants to use a different value (eg, `verify-full`) they need to edit the SQLAlchemy URI directly.